### PR TITLE
Improve English and correct typos

### PR DESCRIPTION
**Pull request by jmaneyrol69**
Improve English and various typos in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
-* Open Genymotioin from vscode
-* Custome Genymotion and VirtualBox path
+* Open Genymotion from vscode
+* Customize Genymotion and VirtualBox paths
 
 ## Getting Started
 1. [Install the extension](https://marketplace.visualstudio.com/items?itemName=Behrad.genymotion)
-2. Run Genymotion sertting from command pallet and detecte Genymotion and VirtualBox path.
-3. Run Genymotion devices command from command pallet and choise your Genymotion devices to start.
+2. Run `Genymotion: Setting` command from *Command pallet* and specify Genymotion and VirtualBox path.
```
Comment (agateau-g) [inline README.md]: I think you can replace `pallet` with `palette` here and on the line below.

Comment (jmaneyrol69) [reply README.md]: Nice catch! Thanks

**Comment by jmaneyrol69** [comment]
Hey @abehrad-ir :wave: 
According to your code (src/PathHelper.ts), it seems your extension does not detect Genymotion and VirtualBox paths for Linux.
Is it correct or am I missing something?

File: README.md [hunk 2]
```diff
@@ -9,18 +9,18 @@
...
-* Custome Genymotion and VirtualBox path
+* Open Genymotion from vscode
+* Customize Genymotion and VirtualBox paths
 
 ## Getting Started
+
 1. [Install the extension](https://marketplace.visualstudio.com/items?itemName=Behrad.genymotion)
-2. Run Genymotion sertting from command pallet and detecte Genymotion and VirtualBox path.
-3. Run Genymotion devices command from command pallet and choise your Genymotion devices to start.
+2. Run `Genymotion: Setting` command from *Command pallette* and specify Genymotion and VirtualBox path.
```
Comment (agateau-g) [inline README.md]: Almost there, but still too many `l` :smile:

Comment (jmaneyrol69) [reply README.md]: Geeze! How careless of me...

**Pull request merged**